### ISSUE TITLE
installing R packages on windows

### ISSUE THREAD

**Issue by LisaMc** [FredHutch/Oncoscape#66]
building R packages may require compilers and other components not available on Windows or bare Linux machines.  Here are some links for obtaining and installing them:

### Developer Instructions
http://www1.appstate.edu/~arnholta/Software/MakingPackagesUnderWindows.pdf

### Rtools for building R and R packages
https://cran.fhcrc.org/bin/windows/Rtools/

**Comment by robert-mcdermott** [comment]
@LisaMc I was able to get oncoscape working on Windows, but it doesn't work out of the box. m4, ps, kill, nohup, egrep, awk, etc are not commands available on Windows so the makefile to run the app doesn't work. I had to run it manually "R --vanilla -f runOncoscapeApp-7777.R"

The Rtools package provides make, gcc, g++, etc... required to compile the R modules from source, but it doesn't provide all the required Unix commands. Rtools uses a subset of Cygwin (https://cygwin.com/) to provide some of the unix tools. Perhaps if we can install the full Cygwin suite and get it working, if not we'll need to make some modifications to get it to run on Windows with the current "make oncoApp7777" method.

Behold Oncoscape on Windows:
>
![image](https://cloud.githubusercontent.com/assets/7399563/10894227/37c9efc6-8162-11e5-91fa-f5a2831b819f.png)
